### PR TITLE
EDX-5340 Add Key data model and DTO

### PR DESCRIPTION
**Pull request by weichou1229**
Add Key data model and DTO

### CODE REVIEW

File: pkg/dtos/keydata.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import (
+	"strings"
+
+	"github.com/IOTechSystems/go-mod-central-ext/v4/pkg/models"
+)
+
+type KeyData struct {
+	Issuer string `json:"issuer" validate:"required"`
+	Type   string `json:"type"`
```
[the remark this answers is not in the document]
Comment (weichou1229) [reply pkg/dtos/keydata.go]: Yes, modified, use `validate:"omitempty,oneof='verification' 'signing'"`, if user didn't input the type, will use verification as default.

File: pkg/dtos/keydata.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+import (
+	"strings"
+
+	"github.com/IOTechSystems/go-mod-central-ext/v4/pkg/models"
+)
+
+type KeyData struct {
+	Issuer string `json:"issuer" validate:"required"`
+	Type   string `json:"type" validate:"omitempty,oneof='verification' 'signing'"`
```
Comment (lindseysimple) [inline pkg/dtos/keydata.go:13]: The single quote is not necessary unless the value contains space. Probably we can remove them.
```suggestion
	Type   string `json:"type" validate:"omitempty,oneof=verification signing"`
```

Comment (weichou1229) [reply pkg/dtos/keydata.go]: Removed.

**Review by lindseysimple — APPROVED** [review]
LGTM

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**